package-lock.json

```diff
@@ -5,29 +5,30 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.5.5",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.5.5.tgz",
-      "integrity": "sha512-27d4lZoomVyo51VegxI20xZPuSHusqbQag/ztrBC7wegWoQ1nLREPVSKSW8byhTlzTKyNE4ifaTA6lCp7JjpFw==",
```

Comment:
This seems unrelated. Can you reset it?

Reply:
On a fresh clone, I was unable to run npm install or npm ci because of issues with this lock file, specifically some semantic-release sub dep that removed a version from npmjs.com.

I'll remove this from this PR, but needs to addressed in general. @gr2m should we update `semantic-release` to ^17.0?